images/bootstrap/Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     python3 \
     python3-distutils \
     python3-pip \
+    python3-yaml \
```

Comment:
@dims I think that pyyaml is available for python3, it just causes issues with pyyaml when using python2. See https://github.com/kubernetes/kubernetes/pull/93627 (this caused `verify-publishing-bot` to pass successfully)

Reply:
Also: https://github.com/kubernetes/test-infra/blob/1a109d935cbfdb62da526e0c5c589cf79ebd5410/requirements3.txt#L8